### ISSUE TITLE
Plots/Charts should load at instant

### ISSUE THREAD

**Issue by eusonlito** [allinurl/goaccess#458]
Plots/Charts load very slow, I think that should load at instant :)

**Comment by allinurl** [comment]
Are you getting a different load time than what the [live demo](http://rt.goaccess.io/?src=gh) shows? Transitions are only shown on the first load and when switching the chart/metric. Real-time updates do not use transitions.

**Comment by eusonlito** [comment]
Yes, but my stats are from previous year, then as bars are load with a small transition, it finish after more than 1 minute.

**Comment by allinurl** [comment]
I changed the bar transition to scale delay dynamically to the length of the dataset. Just shipped on [v1.0.2](https://goaccess.io/). Thanks!

**Comment by eusonlito** [comment]
Now perfect! Thanks!
